test/e2e/generator/generate.go

```diff
@@ -41,7 +40,7 @@ var (
 	// FIXME: v2 disabled due to flake
 	nodeFastSyncs         = uniformChoice{"v0"} // "v2"
 	nodeStateSyncs        = uniformChoice{false, true}
-	nodeMempools          = uniformChoice{"v0", "v1", "v2"}
+	nodeMempools          = uniformChoice{"v0", "v1"}
```

Comment:
we no longer want to test cat in e2e?

Reply:
ahh good spot, sorry, initially I just copy pasted upstream's e2e repo over ours to see the diff and left this guy out

Reply:
we're setting it to v1 elsewhere. the e2e test configures nodes in like 4+ different places.